scripts/build-linux.sh

```diff
@@ -44,8 +44,8 @@ if [ "$FIRMWARE" != "linux" ]; then
 fi
 
 # Install a toolchain with the newlib standard library
-if ! ${CPU_ARCH}-elf-newlib-gcc --version > /dev/null 2>&1; then
```

Comment:
In line 172 we use this compiler to build VexRiscv's emulator. I suggest changing it from:
```
export RISCV_BIN="${CPU_ARCH}-elf-newlib-"
```
to:
```
export RISCV_BIN="${CPU_ARCH}-linux-musl-"
```